meteor/server/api/showStyles.ts

```diff
@@ -132,18 +176,48 @@ export async function removeShowStyleVariant(
 	await ShowStyleVariants.removeAsync(showStyleVariant._id)
 }
 
+export async function reorderAllShowStyleVariants(
+	context: MethodContext,
+	showStyleBaseId: ShowStyleBaseId,
+	orderedVariants: ShowStyleVariant[]
+): Promise<void> {
+	await assertShowStyleBaseAccess(context, showStyleBaseId)
+	await reassignShowStyleVariantIndexes(orderedVariants)
```

Comment:
There are a few problems that should be addressed here:

1. Technically, we haven't checked that the ShowStyleVariants actually exist inside the showStyleBaseId here. We should probably to that.
2. What about if not all of the variants in a ShowStyleBase are provided? This needs to be ensured to avoid settings strange `_rank`s.